bitnami/postgresql-ha/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql-ha
-version: 3.0.1
+version: 3.0.2
```

Comment:
I'd increase the minor version. This is quite a feature to be considered just a pach